src/main/java/com/openshift/jenkins/plugins/pipeline/dsl/OpenShiftBuilder.java

```diff
@@ -91,8 +93,27 @@ public String getWaitTime(Map<String,String> overrides) {
 			return val;
 		return Long.toString(GlobalConfig.getBuildWait());
 	}
-	
-	
+
+	public String getWaitUnits() { return this.waitUnit; }
+
```

Comment:
Also, you'll need a setter for the wait units with an DataBoundSetter annotation for the wait units (those params get set via setters with the dsl, whereas they get set with the constructor for the classic freestyle jobs)